Makefile

```diff
@@ -31,7 +31,7 @@ ifeq ($(HELP),y)
 all: clean
 	@echo "$$ALL_HELP_INFO"
 else
-all: verify-golang clean
```

Comment:
It's no need to clean here, if I make cloudcore first, then I make edgecore, cloudcore will be deleted. cc @kadisi